nfs-ganesha-stable/config/definitions/nfs-ganesha-stable.yml

```diff
@@ -19,6 +19,7 @@
           skip-tag: true
           wipe-workspace: true
           basedir: "nfs-ganesha-debian"
+
```

Comment:
@alfredodeza for some reason the ntirpc scm wasn't being applied here. The spacing was the only reason i can think of. Do you have any guess as to why it wasn't there? I added it in by hand on the jenkins gui to find the rest of the errors fixed in this commit.